airflow/models.py

```diff
@@ -1954,8 +1954,16 @@ class derived from this one results in the creation of a task object,
     template_fields = []
     # Defines wich files extensions to look for in the templated fields
     template_ext = []
-    # Defines the extra buttons to display in the task instance model view
-    extra_links = []
+    # Defines which function to call per button
+    extra_link_functions = {}
```

Comment:
We need to document what the arguments to the link functions will be

Reply:
Added.